tutorials/plot_background_morph.py

```diff
@@ -0,0 +1,875 @@
+# -*- coding: utf-8 -*-
+r"""
+===================================
+Background information on morphing
+===================================
+
+Here we give some background information on morphing in general,
+and how it is done in MNE-Python in particular. Recommended reading and
+corresponding information can be found in Gramfort *et al.* 2013 [1]_ and
+Avants *et al.* 2009 [2]_ as well as in this `dipy example`_. For a shorter and
+version of this tutorial see :ref:`sphx_glr_auto_tutorials_plot_morph.py`.
+
+.. contents::
+    :local:
+
+Problem statement
+=================
+
+Modern neuroimaging techniques such as source reconstruction or fMRI analyses,
+make use of advanced mathematical models and hardware to map brain activity
+patterns into a subject specific anatomical brain space.
+This enables the study of spatio-temporal brain activity. Amongst many others,
+the representation of spatio-temporal activation patterns is often achieved by
+overlaying the anatomical brain structure, with the respective activation at
+the respective anatomical location. Hereby volumetric anatomical MR images are
+often used as such or are transformed into an inflated surface.
+
+It becomes obvious that in order to compute group level statistics, data
+representations across subjects must be morphed to a common frame, such that
+anatomically and functional similar structures are represented at the same
+spatial location.
+
+.. _tut_morphing_basics:
+
+Morphing basics
+================
+
+Morphing describes the procedure of transforming a data representation in n-
+dimensional space into another data representation in the same space. In the
+context of neuroimaging data this space will mostly be 3-dimensional data and
+is necessary to bring individual subject brain spaces into a common frame. Is
+it possible to just shift one brain image such that it is overlaying with a
+second?
+
+Generally speaking yes, but it needs to be kept in mind, that in order to
+achieve a highly accurate overlay, morphological differences have to be
+accounted as well. Not just do sizes of brains vary, but also how cortical and
+subcortical structures are expressed in terms of size and their location
+relative to other structures. Thus it is inevitable to use slightly more
+complex operations than just shifting brains relative to each other.
+Nevertheless, this can be a first step of a successful subject to subject
+transformation.
+
+We can understand mapping as a mathematical function :math:`f`. This function
```

Comment:
not rendered correctly. You may want to build the docs locally and check first before pushing ...